crates/libeq_wld/src/parser/fragments/texture_images.rs

```diff
@@ -58,7 +59,6 @@ impl FragmentParser for TextureImagesFragment {
 
 impl Fragment for TextureImagesFragment {
     fn into_bytes(&self) -> Vec<u8> {
-        let padding = vec![0; 4 - (self.size() % 4)];
```

Comment:
Turns out the fragment body is _always_ padded to a 4 byte boundary. We just ran into the issue here first :P. So this has been moved up to the top level where we can handle all fragments.